client/src/pages/Login.js

```diff
@@ -1,7 +1,106 @@
-import { Box } from "@material-ui/core";
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core'
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import useStyles from '../styles'
+import { useForm } from 'react-hook-form'
+import green from '@material-ui/core/colors/green'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Checkbox from '@material-ui/core/Checkbox'
 
-function Login() {
-  return <Box>Login Page</Box>;
+
+const Login = () => {
+    const classes = useStyles()
+
+    const { register, handleSubmit, formState: { errors }, watch, setValue } = useForm({
```

Comment:
```suggestion
    const { register, handleSubmit, formState: { errors }, setValue } = useForm({
```
This variable is also not being used currently

Reply:
I will remove this variable. thanks